backend/PythonClient/multirotor/airsim_application.py

```diff
@@ -2,14 +2,18 @@
 import math
 import os
 import time
+from pymongo import MongoClient
+from gridfs import GridFS
 from abc import abstractmethod
-
 from PythonClient import airsim
 
 
 class AirSimApplication:
     # Parent class for all airsim client side mission and monitors
     def __init__(self):
+        self.client = MongoClient("mongodb+srv://<db_username>:<db_password>@dronereports.lkmgn.mongodb.net/?retryWrites=true&w=majority&appName=DroneReports")
```

Comment:
It seems wrong to have this URL hard coded (even though it's parameterized with username and password). Perhaps using an environment variable would be better.